packages/runtime/src/runtimes/react/components/hooks/use-builder-connection-ping.ts

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useRef } from 'react'
+import { ActionTypes } from '../../../../react'
+
+const CONNECTION_PING_INTERVAL_MS = 20
+
+export function useBuilderConnectionPing(appOrigin: string) {
+  const sanitizedAppOrigin = appOrigin.replace("'", "\\'")
```

Comment:
I believe this is unnecessary and is an artifact of `appOrigin` previously being interpolated in a string